zerver/management/commands/create_user.py

```diff
@@ -78,7 +83,8 @@ def handle(self, *args, **options):
                 full_name = input("Full name: ")
 
         try:
-            notify_new_user(do_create_user(email, initial_password(email),
+            pw = options.get('password', initial_password(email))
+            notify_new_user(do_create_user(email, pw,
```

Comment:
I guess I'm imagining that this option will in practice primarily be used either in the development environment or in situations like the Docker setup where you just created a container and thus there can't be any other users inside yet.